docs/components/radiobuttons/react.md

```diff
@@ -8,7 +8,14 @@ You must keep track of state yourself. The state has to be handled differently d
 import { Toggle } from '@warp-ds/react';
 ```
 
-## React syntax
+### Syntax
+
+```ts example
+type ToggleEntry = {
+  label: string;
+  value: unknown;
+};
+```
```

Comment:
Is this necessary to add when using Toggle in React?

Reply:
I just moved this section up from line 55, as it made no sense to have it there... 🤷